packages/mobx-state-tree/__tests__/core/array.test.ts

```diff
@@ -560,3 +560,10 @@ test("assigning filtered instances works", () => {
     expect(done.every((t) => !isAlive(t))).toBe(true)
     expect(notDone.every((t) => isAlive(t))).toBe(true)
 })
+
+test("#1676 should accept read-only arrays", () => {
```

Comment:
Awesome, thank you @adamkovalsky!

Mega nitpick: All other issue-fixing tests are denoted by `#issue_num - description`. Could we format this one like this as well?

Reply:
No problem, done!